test/exercises/duplicates/duplicates-alt-assoc-reduce.chpl

```diff
@@ -13,7 +13,7 @@ record Set {
   var keys: domain(keyType);
 }
 
-proc +(a:Set(?t), b:Set(t)) {
+operator Set.+(a:Set(?t), b:Set(t)) {
```

Comment:
I was asked to review the files in this directory, but these are @mppf's work, so I'm re-tagging him in case he wants to take a look.